include/dlaf/matrix/panel.h

```diff
@@ -190,18 +195,68 @@ struct Panel<axis, const T, D> {
     return end_local_;
   }
 
+  /// Set the width of the col panel.
+  ///
+  /// By default the width of the panel is parentDistribution().block_size().cols().
+  /// This method allows to reduce this value.
+  ///
+  /// @pre this can be called as first operation after range setting.
+  /// @pre @param 0 < width <= parentDistribution().block_size().cols()
+  template <Coord A = axis, std::enable_if_t<A == axis && Coord::Col == axis, int> = 0>
+  void setWidth(SizeType width) noexcept {
+    DLAF_ASSERT(width > 0, width);
+    DLAF_ASSERT(width <= parentDistribution().blockSize().cols(), width,
+                parentDistribution().blockSize().cols());
+
+    dim_ = width;
+  }
+
+  /// Set the height of the row panel.
+  ///
+  /// By default the  oghtf the panel is parentDistribution().block_size().rows().
```

Comment:
```suggestion
  /// By default the height the panel is parentDistribution().block_size().rows().
```
?

Reply:
Fixed by https://github.com/eth-cscs/DLA-Future/pull/391/commits/c617901b9dd088835bb84e8599affc70e098b1ab in #391

Reply:
Needs a rebase?